lib/utils/hfbBuffer.cpp

```diff
@@ -0,0 +1,248 @@
+#include "hfbBuffer.hpp"
+
+#include "gpsTime.h"
+
+#include "fmt.hpp"
+
+#include <set>
+
+
+template<typename T>
```

Comment:
These two are obviously shared with `visFrameView` and so we should move to a common location. It sounds like you're on top of that one though.